_posts/2020-06-15-quarkus-hackathon-announcement.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: 'Quarkus Global Hackathon'
+date: 2020-06-15
+tags: announcement
+synopsis: Quarkus global hackathon is Live!
+author: jbeck
+---
+:imagesdir: /assets/images/posts/announcements
+
+= Quarkus Global Hackathon
+
+image::quarkushackathon_header.png[]
+
+The Quarkus community is excited to announce the https://quarkus.devpost.com/[Supersonic, Subatomic Java Hackathon] for developers to create Kubernetes-native applications for a chance to win *$30,000* in prizes. This hackathon is a great opportunity to learn about the future of cloud-native Java development and showcase your coding skills.
+
+The Quarkus community and ecosystem have experienced tremendous growth since it’s inception.  The goal of the hackathon is to continue to grow this ecosystem by showcasing the framework and the developers who power it.  
```

Comment:
```suggestion
The Quarkus community and ecosystem have experienced tremendous growth since its inception.  The goal of the hackathon is to continue to grow this ecosystem by showcasing the framework and the developers who power it.  
```